src/applications/hca/tests/unit/components/FormPages/VeteranInformation.unit.spec.js

```diff
@@ -56,86 +56,58 @@ describe('hca VeteranInformation', () => {
       dispatch: () => {},
     },
   });
+  const subject = ({ mockStore, props }) => {
```

Comment:
I love how much code using these subjects with the selectors simplifies 👍

Reply:
Same here!! I didn't quite understand when I first watched you implement it.. but it really does clean these tests up a ton!